Installation-CentOS-8-Apache

```diff
@@ -0,0 +1,356 @@
+location.
+
+```
+module http_fping 1.0;
+
+require {
+type httpd_t;
+class capability net_raw;
+class rawip_socket { getopt create setopt write read };
+}
+
+#============= httpd_t ==============
+allow httpd_t self:capability net_raw;
+allow httpd_t self:rawip_socket { getopt create setopt write read };
+```
+
+Then run these commands
+
+```
+checkmodule -M -m -o http_fping.mod http_fping.tt
+semodule_package -o http_fping.pp -m http_fping.mod
+semodule -i http_fping.pp
+```
+
+# Allow access through firewall
+
+```
+firewall-cmd --zone public --add-service http
+firewall-cmd --permanent --zone public --add-service http
+firewall-cmd --zone public --add-service https
+firewall-cmd --permanent --zone public --add-service https
+```
+
+# Configure snmpd
+
+Copy the example snmpd.conf from the LibreNMS install.
+
+```
+cp /opt/librenms/snmpd.conf.example /etc/snmp/snmpd.conf
+```
+
+```
+vi /etc/snmp/snmpd.conf
+```
+
+Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community string.
+
+```
+curl -o /usr/bin/distro https://raw.githubusercontent.com/librenms/librenms-agent/master/snmp/distro
+chmod +x /usr/bin/distro
+systemctl enable snmpd
+systemctl restart snmpd
+```
+
+# Copy logrotate config
+
+LibreNMS keeps logs in `/opt/librenms/logs`. Over time these can
+become large and be rotated out.  To rotate out the old logs you can
+use the provided logrotate config file:
+
+```
+cp /opt/librenms/misc/librenms.logrotate /etc/logrotate.d/librenms
+```
+
+# Web installer
+
+Now head to the web installer and follow the on-screen instructions.
+
+<http://librenms.example.com/install.php>
+
+The web installer might prompt you to create a `config.php` file in
+your librenms install location manually, copying the content displayed
+on-screen to the file. If you have to do this, please remember to set
+the permissions on config.php after you copied the on-screen contents
+to the file. Run:
+
+```
+chown librenms:librenms /opt/librenms/config.php
+```
+
+It will tell you to go to your validate your install by going to <http://librenms.example.com/validate>, fix any issues you might find here.
+
+The validation process might ask you to enter a config value for `fping`. Add the following line to config.php
+
+```
+$config['fping6'] = '/usr/sbin/fping';
+```
+Now will also be the best time to install the service file to run LibreNMS with.
+
+Start off by copying the example `librenms.service` file to the correct path:
+
+```
+cp /opt/librenms/misc/librenms.service /etc/systemd/system/librenms.service
+```
+Insert the following settings into your `config.php` file in order to run the Dispatcher Service
+
+```
+$config['service_poller_workers']              = 16;     # Processes spawned for polling
+$config['service_services_workers']            = 8;      # Processes spawned for service polling
+$config['service_discovery_workers']           = 8;     # Processes spawned for discovery
+
+
+//Optional Settings
```

Comment:
as i know those settings are documented plattform independend else, so i think it's not needed here

Reply:
i think install documentation should only handle such steps to get it to run and no additional/fancy stuff :-)

Reply:
Yes, agree. The reason they are here is because I did not include the cronjob stuff. That is why I asked about which "scheduling method" would be preferable for the forseeable future. As a user I would not want to set up cron during install, just for everyone to tell me to use Dispatcher Service rather than cron. So a decision needs to be made. If it is dispatcher service, this part needs to stay, or they won't have an automated working system at all.